packages/framework-core/src/services/graphql/graphql-query-generator.ts

```diff
@@ -37,12 +38,18 @@ export class GraphQLQueryGenerator {
   public generate(): GraphQLObjectType {
     const byIDQueries = this.generateByIDQueries()
     const filterQueries = this.generateFilterQueries()
+    const fields = {...byIDQueries, ...filterQueries}
+    if (Object.keys(fields).length === 0) {
+      return new GraphQLObjectType({
+        name: 'Query',
+        fields: {
+          _dummy: { type: GraphQLString }
+        }
+      })
+    }
```

Comment:
It is mandatory to always return a query, even though is a silly query. Empty queries violates GraphQL validations

Reply:
Perhaps we can give it a meaningful name? Like `NoQueriesDefined`?

Reply:
I'll change its name accordingly